src/react-apps/applications/shared/src/utils/returnElementListFromString.ts

```diff
@@ -0,0 +1,5 @@
+import ReactHtmlParser from 'react-html-parser';
+
+export function returnElementListFromString(elementString: string) {
```

Comment:
Why is this a shared util?

Reply:
file deleted, and moved this functionality into files that needed it.